app/src/main/java/org/mozilla/fenix/home/mozonline/PrivacyContentDisplayHelper.kt

```diff
@@ -44,15 +47,19 @@ fun showPrivacyPopWindow(context: Context, activity: Activity) {
     // Users can only use fenix after they agree with the privacy notice
     val builder = AlertDialog.Builder(activity)
         .setPositiveButton(
-            context.getString(R.string.privacy_notice_positive_button),
-            { _, _ ->
-                context.settings().shouldShowPrivacyPopWindow = false
+            context.getString(R.string.privacy_notice_positive_button)
+        ) { _, _ ->
+            context.settings().shouldShowPrivacyPopWindow = false
+            context.settings().preferences.edit {
+                putBoolean(context.getString(R.string.pref_key_marketing_telemetry), true)
             }
-        )
+            if (context.settings().isMarketingTelemetryEnabled) {
+                context.components.analytics.metrics.start(MetricServiceType.Marketing)
+            }
+        }
         .setNeutralButton(
-            context.getString(R.string.privacy_notice_neutral_button_2),
-            { _, _ -> exitProcess(0) }
-        )
+            context.getString(R.string.privacy_notice_neutral_button_2)
+        ) { _, _ -> exitProcess(0) }
```

Comment:
nit: Change not needed to fix the issue. Cleanups could be part of separate commits to keep the changes small and easy to follow.